app/Http/Services/NcpoService.php

```
@@ -57,46 +57,71 @@ private function fallback($primary, $fallback)
+                        'unit_price' => number_format($this->fallback($change->changed_unit_price, $original['unit_price']), 2),
+                        'total_amount' => number_format($this->fallback($change->changed_qty, $original['quantity'] ?? 0)
+                            * $this->fallback($change->changed_unit_price, $original['unit_price'] ?? 0), 2),
+                        'net_vat' => number_format($this->fallback($change->net_vat, $original['net_vat']), 2),
+                        'input_vat' => number_format($this->fallback($change->input_vat, $original['input_vat']), 2),
+                        'supplier_name' => $this->getSupplierDetails($purchaseOrder, $change)['changed']['name'] ?? $originalSupplier['name'],
+                        'supplier_address' => $this->getSupplierDetails($purchaseOrder, $change)['changed']['address'] ?? $originalSupplier['address'],
+                        'supplier_contact_number' => $this->getSupplierDetails($purchaseOrder, $change)['changed']['contact_number'] ?? $originalSupplier['contact_number'],
+                    ];
+                } else {
+                    $result['changed'] = 'Pending Approval';
+                }
```

PHPMD (2.15.0)

The method getItemsWithChanges uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)